### PR TITLE
RIA-2045 notification of uploaded additional evidence

### PR DESCRIPTION
**Pull request by tarun-palisetty**
### JIRA link (if applicable) ###
[RIA-2045](https://tools.hmcts.net/jira/browse/RIA-2045)

### Change description ###
Notification of uploaded additional evidence

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/BasePersonalisationProvider.java [hunk 1]
```diff
@@ -152,6 +152,18 @@ public static void buildHearingRequirementsFields(AsylumCase asylumCase, Builder
             .build();
     }
 
+    public Map<String, String> getUploadAdditionalEvidencePersonalisation(AsylumCase asylumCase) {
```
Comment (raghera) [inline src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/BasePersonalisationProvider.java]: Looks like this is quite a specific method for uploadAdditionalEvidence.  Are we sure we want to put this in a Base class?  If we keep doing that then the class will eventually become gigantic.

File: src/main/resources/application-email-addresses.yaml [hunk 2]
```diff
@@ -28,9 +28,10 @@ hearingCentreTelephoneNumber:
   hattonCross: ${IA_HEARING_CENTRE_HATTON_CROSS_TELEPHONE:0300 123 1711}
   glasgow: ${IA_HEARING_CENTRE_GLASGOW_TELEPHONE:0300 123 1711}
 
-endAppealHomeOfficeEmailAddress: ${IA_HOME_OFFICE_END_APPEAL_EMAIL:ho-end-appeal@example.com}
-allowedAppealHomeOfficeEmailAddress: ${IA_HOME_OFFICE_ALLOWED_APPEAL_EMAIL:ho-allowed-appeal@example.com}
-dismissedAppealHomeOfficeEmailAddress: ${IA_HOME_OFFICE_DISMISSED_APPEAL_EMAIL:ho-dismissed-appeal@example.com}
+endAppealHomeOfficeEmailAddress: ${IA_HOME_OFFICE_END_APPEAL_EMAIL:ReformpbAPC@example.com}
```
Comment (bradsorour) [inline src/main/resources/application-email-addresses.yaml]: @tarun-palisetty - I think we need to keep the email addresses different to what's in production. I had this discussion with Ravi last week and we effected these changes in ticket RIA-2366. The idea is that it's not easy for anyone to guess what the actual email addresses would be production, simply by looking at the code (publicly visible) within GitHub.

Comment (tarun-palisetty) [reply src/main/resources/application-email-addresses.yaml]: modified the email accordingly

File: src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/BasePersonalisationProvider.java (hunk 1 above)
Comment (raghera) [reply src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/BasePersonalisationProvider.java]: Decided to refactor this later in a more considered way.  Better to get this delivered now for business value.

**Pull request merged**